crates/voicevox_core/src/voice_model.rs

```diff
@@ -149,23 +178,23 @@ impl VvmEntryReader {
     async fn read_vvm_entry(&self, filename: &str) -> Result<Vec<u8>> {
         let me = self.entry_map.get(filename).ok_or(Error::VvmRead {
             filename: filename.into(),
-            source: None,
+            source: None, // FIXME: ちゃんとエラーを返す
```

Comment:
正しくVVMファイルが作れていなくて、存在しないファイルを読もうとしてここで落ちていましたが、asyncなこととエラメッセージがなかったことでかなり特定に苦戦しました。
ここにちゃんとエラー情報格納したいのですが、どういうコードを書けば良いのかわからないかったのでそのままにしています。

Reply:
```suggestion
            // use anyhow::anyhow;
            // ...
            source: Some(anyhow!("VVMファイル内にファイルが存在しません")),
```
こんな感じでしょうか？（こういうエラーメッセージで合ってるかはわかりませんが）

Reply:
ありがとうございます！！anyhow、なるほどです。
zipエラーはそうだとわかるようにしつつ、そもそもどのファイルがエラーになったのかわからないようになってたのでエラーに含めるようにしてみました！